azure-self-hosted-runners/post-deployment-script.ps1

```diff
@@ -1,25 +1,24 @@
 param (
-    # GitHub Actions Runner registration token. Note that these tokens are only valid for one hour after creation, so we always expect the user to provide one.
     # https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory = $true, HelpMessage = "GitHub Actions Runner registration token. Note that these tokens are only valid for one hour after creation, so we always expect the user to provide one.")]
     [string]$GitHubActionsRunnerToken,
 
     # GitHub Actions Runner repository. E.g. "https://github.com/MY_ORG" (org-level) or "https://github.com/MY_ORG/MY_REPO" (repo-level)
     # https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners
     [Parameter(Mandatory = $true)]
+    [ValidateScript({ $_ -notlike "https://*" })]
```

Comment:
Should this be `-like` instead? We expect this to be a HTTPS URL, right?

Reply:
That's correct I'll change it